Medium/sample.ipynb

```diff
@@ -0,0 +1,296 @@
+   "id": "5d7b0c0b-5e6c-4be3-b901-704fc00b6c08",
+   "metadata": {
+    "papermill": {},
+    "tags": []
+   },
+   "source": [
+    "# Medium - Publish article from Mardown file"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "ba4944a7-f4e5-4e89-9e3e-b4ee2e12c15f",
+   "metadata": {
+    "papermill": {},
+    "tags": []
+   },
+   "source": [
+    "**Tags:** #medium #publish #article #markdown #file #api"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "c96fc2a3-0910-4b6d-a735-1e83cd5fa24e",
+   "metadata": {
+    "papermill": {},
+    "tags": []
+   },
+   "source": [
+    "**Author:** [SaiKiran M](www.linkedin.com/in/msaikiran9)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "edcf9f6c-c073-46c4-92ab-cc72866daf37",
+   "metadata": {
+    "papermill": {},
+    "tags": []
+   },
+   "source": [
+    "**Last update:** 2023-10-09 (Created: 2023-10-09)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "9b1586dd-d9e8-4f75-a583-c5cd5952bb05",
+   "metadata": {
+    "papermill": {},
+    "tags": []
+   },
+   "source": [
+    "**Description:** This notebook explains how to publish an article from a Markdown file via the Medium API. It is usefull for organizations that need to quickly publish content on Medium."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "cc1d8d75-0c1f-41ad-a6b0-b0eb43fcc665",
+   "metadata": {
+    "papermill": {},
+    "tags": []
+   },
+   "source": [
+    "**References:**\n",
+    "- [Programmatically Publish a Markdown File as a Medium Story with Python](https://betterprogramming.pub/programmatically-publish-a-markdown-file-as-a-medium-story-with-python-b2b072a5f968)\n",
+    "- [Medium API Documentation](https://github.com/Medium/medium-api-docs)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "32f771ab-b85d-44c2-8555-004b07b512ea",
+   "metadata": {
+    "papermill": {},
+    "tags": []
+   },
+   "source": [
+    "## Input"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "74c2e136-cd6b-4079-9338-0b609b927692",
+   "metadata": {
+    "papermill": {},
+    "tags": []
+   },
+   "source": [
+    "### Import libraries"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "e4bad8c4-13c7-4c1c-b97d-46fc1ced422c",
+   "metadata": {
+    "papermill": {},
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "try:\n",
+    "    import naas\n",
+    "    import json\n",
+    "    import requests\n",
+    "except ImportError:\n",
+    "    print(\"One or more modules not found. Installing necessary packages...\")\n",
+    "    !pip install requests\n",
+    "    import requests"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "85e28cc8-0742-4aed-a9ee-c0b98cc00de4",
+   "metadata": {
+    "papermill": {},
+    "tags": []
+   },
+   "source": [
+    "### Setup variables\n",
+    "- `token`: Medium access token. [Instructions to get the token](https://github.com/Medium/medium-api-docs#22-authentication).\n",
```

Comment:
![image](https://github.com/jupyter-naas/awesome-notebooks/assets/81156510/d9d9a9fe-ad94-42a1-be79-97a6a1556d12)

A step-wise guide would be better. Something similar to take inspiration from, not the exact same thing.